src/rpc/blockchain.cpp

```diff
@@ -47,7 +47,6 @@
 
 #include <boost/thread/thread.hpp> // boost::thread::interrupt
 
-#include <mutex>
```

Comment:
16325: changes in `include`s do not match. also, missing changes in `test/functional/wallet_resendwallettransactions.py`.

Reply:
For header file changes 
we cant remove this **#include <key_io.h>** yet, as  this will cause build to break

In file included from ./script/descriptor.h:9,
                 from rpc/blockchain.cpp:24:
./script/sign.h:14:7: note: forward declaration of ‘class CKey’
   14 | class CKey;

To remove it we need to implement Merge #16227: Refactor CWallet's inheritance chain 

Also moving of #include <condition_variable> header from line 47 to 44 in original change looks different in our dash code as we don't have line #include <memory> in line 45 (cant pull this changes as this is marked as DNM)  https://github.com/bitcoin/bitcoin/pull/16325/files src/rpc/blockchain.cpp
So header changes are looking different

Somehow I missed this test/functional/wallet_resendwallettransactions.py I have added it now
I have marked 16325 as partial Merge we can remove key_io once Merge #16227 is merged